util/load-soak-testing/bin/basicalicebob.rs

```diff
@@ -0,0 +1,138 @@
+use anyhow::{Context, Result};
+use aptos_sdk::{
+	coin_client::CoinClient,
+	rest_client::{Client, FaucetClient},
+	types::LocalAccount,
+};
+use load_soak_testing::execute_test;
+use load_soak_testing::init_test;
+use load_soak_testing::ExecutionConfig;
+use load_soak_testing::Scenario;
+use std::str::FromStr;
+use url::Url;
+
+fn main() {
+	// Define the Test config. Use the default parameters.
+	let config = ExecutionConfig::default();
+
+	// Init the Test before execution
+	if let Err(err) = init_test(&config) {
+		println!("Test init fail ; {err}",);
+	}
+
+	// Execute the test.
+	let result = execute_test(config, &create_scenario);
+	tracing::info!("End Test with result {result:?}",);
+}
+
+// Scenario constructor function use by the Test runtime to create new scenarios.
+fn create_scenario(id: usize) -> Box<dyn Scenario> {
+	Box::new(BasicScenario { id })
+}
+
+pub struct BasicScenario {
+	id: usize,
+}
+
+impl BasicScenario {
+	pub fn new(id: usize) -> Self {
+		BasicScenario { id }
+	}
+}
+
+// Scenario trait implementation.
+#[async_trait::async_trait]
+impl Scenario for BasicScenario {
```

Comment:
I sort of get this but maybe it would help to describe what the scenario is exactly